app/src/main/java/com/android/streetworkapp/ui/park/ParkOverview.kt

```diff
@@ -392,26 +396,40 @@ fun EventItemList(eventViewModel: EventViewModel, navigationActions: NavigationA
  */
 @Composable
 fun EventItem(event: Event, eventViewModel: EventViewModel, navigationActions: NavigationActions) {
+  val (statusColor, statusIcon) =
+      when (event.status) {
+        EventStatus.CREATED -> Pair(Color.Red, Icons.Default.Timelapse)
```

Comment:
UI: the green and red color are a little too different from the rest of the UI. For the red, I suggest that we use the same color as the one for the picture. (you can see it in PR#179 , the last screenshot)

For the green, I do not think that we already have a instance of green in our UI, but I think that making it a darker green could be better for the UI.

Reply:
(I just realized: I think that the best possible green is the one that is used in this page. Look att the "Open" logo, just under the title of the PR. I think this is the best for our UI)